packages/dialogs/src/RPA/Dialogs/library.py

```diff
@@ -696,6 +697,53 @@ def add_drop_down(
 
         self.add_element(element)
 
+    @keyword("Add date-picker", tags=["input"])
+    def add_date_picker(
+            self,
+            name: str,
+            default: Optional[str] = None,
+            label: Optional[str] = None,
+    ) -> None:
+        """Add a date-picker element
+
+        :param name:    Name of result field
+        :param default: The default date
+        :param label:   Label for input field
+
+        Displays a date picker widget. The selection the user makes will be available
+        in the ``name`` field of the result.
+        The ``default`` argument can be a pre-set date in DD/MM/YYYY format, otherwise
+        the current date is used.
+        A custom ``label`` text can also be added.
+
+        Example:
+
+        .. code-block:: robotframework
+
+            Add heading     Enter your birthdate
+            Add date-picker
+            ...    name=birthdate
+            ${result}=      Run dialog
+            Log    User birthdate should be: ${result.birthdate}
+        """
+        format = "%d/%m/%Y"
+        if default:
+            try:
+                datetime.strptime(default, format)
+            except Exception as exc:
+                raise ValueError(f"Invalid default date with value {default!r}") from exc
+        else:
+            default = datetime.utcnow().strftime(format)
+
+        element = {
+            "type": "input-datepicker",
```

Comment:
We're blocked here by this other robocorp-dialogs [PR](https://github.com/robocorp/robocorp-dialog/pull/1).

Reply:
Not blocked anymore.